rust/Cargo.toml

```diff
@@ -44,15 +33,17 @@ crate-type = ["staticlib"]
 [profile.release]
 lto = true
 
-# Revision corresponds to the partially-complete auto-shielding version, with the NU5
-# consensus branch ID and activation heights, and v5 transaction parsing, backported.
-# https://github.com/zcash/librustzcash/pull/555
-# https://github.com/zcash/librustzcash/pull/558
+# Revision corresponds to zcash_primitives 0.6.0 plus the NU5 mainnet activation height.
+# We also need zcash_client_backend and zcash_client_sqlite, which haven't been published
+# with NU5 updates yet.
 [patch.crates-io]
-zcash_client_backend = { git = 'https://github.com/zcash/librustzcash.git', rev='0a1ed9b8acf336bdc20ba49083c66f114ecf6877' }
-zcash_client_sqlite = { git = 'https://github.com/zcash/librustzcash.git', rev='0a1ed9b8acf336bdc20ba49083c66f114ecf6877' }
-zcash_primitives = { git = 'https://github.com/zcash/librustzcash.git', rev='0a1ed9b8acf336bdc20ba49083c66f114ecf6877' }
-zcash_proofs = { git = 'https://github.com/zcash/librustzcash.git', rev='0a1ed9b8acf336bdc20ba49083c66f114ecf6877' }
+zcash_address = { git = 'https://github.com/zcash/librustzcash.git', rev='6047c1d0030b2a40abd35cbc7ba15bd2bd63fc8d' }
```

Comment:
I think these patches should be removed and point to creates only

Reply:
Yeah, @str4d mentioned this in our standup, so I’m already making that change.

Reply:
I’m going to do this in a separate PR.

Reply:
I’m working on this PR right now, _should_ be up for review tonight.